### PR TITLE
[6414] Changed tests to use 2048 key size

### PR DESCRIPTION
**Pull request by JustinKyleJames**

### CODE REVIEW

**Review by korydraughn — APPROVED** [review]

File: scripts/irods/test/resource_suite.py [hunk 1]
```diff
@@ -297,9 +297,9 @@ def test_ssl_iput_with_rods_env(self):
         chain_pem_path = os.path.join(self.admin.local_session_dir, 'chain.pem')
         dhparams_pem_path = os.path.join(self.admin.local_session_dir, 'dhparams.pem')
 
-        lib.execute_command(['openssl', 'genrsa', '-out', server_key_path, '1024'])
+        lib.execute_command(['openssl', 'genrsa', '-out', server_key_path, '2048'])
         lib.execute_command('openssl req -batch -new -x509 -key %s -out %s -days 365' % (server_key_path, chain_pem_path))
-        lib.execute_command('openssl dhparam -2 -out %s 1024' % (dhparams_pem_path))  # normally 2048, but smaller size here for speed
+        lib.execute_command('openssl dhparam -2 -out %s 2048' % (dhparams_pem_path))  # normally 2048, but smaller size here for speed
```
Comment (alanking) [inline scripts/irods/test/resource_suite.py:302]: The comment at the end of this line can be removed.

File: scripts/irods/test/resource_suite.py [hunk 2]
```diff
@@ -362,9 +362,9 @@ def test_ssl_iput_small_and_large_files(self):
...
```
Comment (alanking) [inline scripts/irods/test/resource_suite.py:367]: The comment at the end of this line can be removed.

**Review by alanking — APPROVED** [review]
Please #, yes. Comments optional because comments.

**Comment by alanking** [comment]
Ah, thanks for doin the comments. Let's go ahead and add #'s.

**Review by trel — APPROVED** [review]

**Pull request merged**